toolbox/nfd/has_gpu_nodes.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
```

Comment:
Why are we sourcing if we are not calling any helper func?

Reply:
`_common.sh` is a prolog of the script, 
it sets some ansible variables (inventory, logging) and `cd` to the top directory
https://github.com/openshift-psap/ci-artifacts/blob/master/toolbox/_common.sh